packages/wdio-cli/tests/launcher.test.js

```diff
@@ -1,10 +1,12 @@
 import Launcher from '../src/launcher'
 import logger from '@wdio/logger'
-import fs from 'fs-extra'
-import path from 'path'
 
 const caps = { maxInstances: 1, browserName: 'chrome' }
 
+jest.mock('fs-extra')
+const fs = require('fs-extra')
+let ensureDirSyncSpy
```

Comment:
let's move the variable declaration (`let ensureDirSyncSpy`) into the `describe('config options', () => {` block

Reply:
cool I ll do that

Reply:
Updated for both tests